app/client/src/components/Post.jsx

```diff
@@ -95,7 +88,7 @@ const Post = (props) => {
   return (
     <div
```

Comment:
Instead of using <div>, use <Box> from mui. The <Box> component is essentially just an mui div and it supports the use of the 'sx' prop with which the 'style' prop can be replaced.